composer.json

```diff
@@ -11,7 +11,11 @@
         }
     ],
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "guzzlehttp/psr7": "^1.0",
+        "lstrojny/fxmlrpc": "^0.14.0",
+        "php-http/client-implementation": "^1.0",
```

Comment:
You don't need this neither the line below.

This is `lstrojny/fxlrpc` dependencies.

BTW, this should fix this issue: https://github.com/nexylan/gandi-sdk/pull/1/files#r143978680